test/runHWTests.m

```diff
@@ -0,0 +1,103 @@
+function runHWTests(board)
+
+import matlab.unittest.TestRunner;
+import matlab.unittest.TestSuite;
+import matlab.unittest.plugins.TestReportPlugin;
+import matlab.unittest.plugins.XMLPlugin
+import matlab.unittest.plugins.DiagnosticsValidationPlugin
+import matlab.unittest.parameters.Parameter
+import matlab.unittest.plugins.ToUniqueFile;
+import matlab.unittest.plugins.TAPPlugin;
+import matlab.unittest.constraints.ContainsSubstring;
+import matlab.unittest.selectors.HasName;
+import matlab.unittest.selectors.HasProcedureName;
+
+switch board
```

Comment:
The boards and tests referred to, here, aren't relevant to the Precision Toolbox. We can scrap those cases, perhaps? Also, we would need to add a case and point to LLDKTests.